### ISSUE TITLE
plugin does not work if jquery is loaded again

### ISSUE THREAD

**Issue by steveswinsburg** [jquery-i18n-properties/jquery-i18n-properties#45]
I have a setup where multiple portlets are rendered inline into the same page. Each of these emits the jquery library as it has its own requirements.

One of these uses i18n-properties. Unless it is the last one to be rendered, the subsequent loading of jquery overwrites the i18n registration.

I end up with something like this:
````
<script type="text/javascript" src="/library/webjars/jquery/1.11.3/jquery.min.js"></script>
...
<script type="text/javascript" src="/library/webjars/jquery/1.11.3/jquery.min.js"></script>
<script type="text/javascript" src="/my-calendar/scripts/jquery.i18n.properties.min.js"></script>
...
<script type="text/javascript" src="/library/webjars/jquery/1.11.3/jquery.min.js"></script>
````

This then causes:
`jQuery.i18n is undefined`

If the webapp using i18n-properties is the last one to be rendered, then it works, but this is not guaranteed as it is a portal.

I am wondering if this could be resolved via namespacing the invocation of i18n-properties in the specific portlet? I currently load it like this:

````
$(document).ready(function() {

	jQuery.i18n.properties({
	    name:'Messages', 
	    path:'/my-calendar/bundle/',
	    mode: 'both',
	    cache: true
	});

...
````

**Comment by steveswinsburg** [comment]
Ok this appears to be a namespacing issue. I have removed i18n.properties and I now get conflicts on other libraries.

**Comment by steveswinsburg** [comment]
OK I will close this off. The problem was that jquery was being loaded into the `<head>` by each portlet after the i18n was loaded into the `<head>` as well (each portlet contributes to the html head in turn). This was breaking the plugins. I moved the i18n and other plugins out of the html head and next to the code where it is being used and left jQuery in the head (there were some other requirements which meant it needed to be in the head and not within the page. This has resolved the issue.

TL;DR: PEBKAC!
